app/pages/lab/project.cjsx

```diff
@@ -82,24 +93,28 @@ EditProjectPage = React.createClass
           <li>
             <br />
             <div className="nav-list-header">Workflows</div>
-            <PromiseRenderer promise={@props.project.get 'workflows'}>{(workflows) =>
-              <ul className="nav-list">
-                {renderWorkflowListItem = (workflow) ->
-                  <li key={workflow.id}>
-                    <Link to={@labPath("/workflow/#{workflow.id}")} activeClassName="active" className="nav-list-item" title="A workflow is the sequence of tasks that you’re asking volunteers to perform.">{workflow.display_name}</Link>
-                  </li>}
+            <PromiseRenderer promise={getWorkflowsInOrder @props.project}>{(workflows) =>
+              renderWorkflowListItem = (workflow) =>
+                <li key={workflow.id}>
+                  <Link to={@labPath "/workflow/#{workflow.id}"} className="nav-list-item" activeClassName="active">
+                    {workflow.display_name}
+                    {if workflow.id is @props.project.configuration?.default_workflow
+                      <span title="Default workflow">{' '}*{' '}</span>}
+                  </Link>
+                </li>
 
-                {for workflow in workflows
-                  <ChangeListener key={workflow.id} target={workflow} eventName="save" handler={renderWorkflowListItem.bind this, workflow} />}
+              renderWorkflowListItemListener = (workflow) =>
+                <ChangeListener key={workflow.id} target={workflow} eventName="save" handler={renderWorkflowListItem.bind null, workflow} />
 
-                <li className="nav-list-item">
-                  <button type="button" onClick={@showCreateWorkflow} disabled={@state.workflowCreationInProgress} title="A workflow is the sequence of tasks that you’re asking volunteers to perform.">
-                    New workflow{' '}
-                    <LoadingIndicator off={not @state.workflowCreationInProgress} />
-                  </button>
-                </li>
-              </ul>
+              <DragReorderable tag="ul" className="nav-list" items={workflows} render={renderWorkflowListItemListener} onChange={@handleWorkflowReorder} />
```

Comment:
Ah, I think I got it now. I guess ChangeListeners cannot be the direct children of a DragReorderable.